pkgs/development/python-modules/torchvision/bin.nix

```diff
@@ -61,7 +61,7 @@ in buildPythonPackage {
     # https://www.intel.com/content/www/us/en/developer/articles/license/onemkl-license-faq.html
     license = licenses.bsd3;
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
-    platforms = platforms.linux;
+    platforms = [ "x86_64-linux" ];
```

Comment:
```suggestion
    platforms = [ "aarch64-darwin" "x86_64-darwin" "x86_64-linux" ];
```

Reply:
patchelf doesn't work on Darwin, so the package as is doesn't support Darwin

Reply:
Ah, I went from the list of hashes in `binary-hashes.nix`.